guides/source/configuring.md

```diff
@@ -3230,6 +3230,14 @@ configured without modifying the URL set in the deployment environment. See:
 
 TIP: You don't have to update the database configurations manually. If you look at the options of the application generator, you will see that one of the options is named `--database`. This option allows you to choose an adapter from a list of the most used relational databases. You can even run the generator repeatedly: `cd .. && rails new blog --database=mysql`. When you confirm the overwriting of the `config/database.yml` file, your application will be configured for MySQL instead of SQLite. Detailed examples of the common database connections are below.
 
+By default, Rails will use the standard CLI interface for your database when running `rails dbconsole` (e.g. `psql` for PostgreSQL and `mysql` for MySQL). If you wish to use an alternative application, you can specify this with the `database_cli` ActiveRecord configuration option. To specify a fallback, you can use an array:
```

Comment:
Instead of adding this here can we create a new section for this new config now like all the others?

https://github.com/rails/rails/blob/73608ff4110710d1dba81da9cd318440d42a9d23/guides/source/configuring.md#configuring-active-record

Reply:
That’s done—sorry I missed the obvious place :facepalm: